modules/acquia_cms_dam/acquia_cms_dam.module

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file
+ * Contains hook implementations for ACMS DAM.
+ */
+
+/**
+ * Implements hook_modules_installed().
+ */
+function acquia_cms_dam_modules_installed($modules) {
```

Comment:
Logic is repeated in this hook!!
@rajeshreeputra can we optimise logic created in this hook?
something like switch statements or create another method where you can put 
`// Update event image field to support Acquia DAM.
  if ($moduleHandler->moduleExists($module_name)) {
    $fieldConfig = \Drupal::configFactory()->getEditable($module_image_config);
    if ($fieldConfig) {
      _acquia_cms_dam_update_image_field($fieldConfig);
    }
  }`

Reply:
Updated